index.html

```diff
@@ -8,43 +9,43 @@
       <h1>ColoRandom</h1>
       <section class="current-palette">
         <div id="color1">
-          <div class="box" id="box1">
+          <div class="box box-1" id="box1">
```

Comment:
I do want to revisit this naming convention. I'm not sure that two classes with such similar names is what we want!